plugin.json

```diff
@@ -2,7 +2,8 @@
   "id": "mattermost-autolink",
   "name": "Autolink",
   "description": "Automatically rewrite text matching a regular expression into a Markdown link.",
-  "version": "1.1.0",
+  "version": "1.1.1",
+  "min_server_version": "5.15.0",
```

Comment:
Out of interest: Why is this needed?

Reply:
/autolink command saves the config, causing OnConfigChanged. In older versions and older bundled plugins there were known deadlocks. We really should be requiring 5.16 now where the deadlocks and other related bugs are fixed, but we can't yet.

Reply:
Why can't we requiring 5.16?

Reply:
Because chicken and egg? If we publish this release of Autolink now, it'd be requiring a non-existing Mattermost version for 2 weeks?

Reply:
The required server version is compared to `model.CurrentVersion`, which was bumped in https://github.com/mattermost/mattermost-server/pull/12172. We can safely bump the version here.

Please also note that `master` server is using `5.18`, hence you can bump the required version to `5.18`, if your plugin relies on something that needs 5.18.

Reply:
Right, but none of these are _released_ and available for download by the customers, yet. Right?

Reply:
That is correct. I think it comes down to what "requires" means. If you highly encourage people to use `5.16` because of bug fixes, do you want to enforce this?

I'm 1/5 that it's fine to release a plugin for a non existing release. I suppose people won't jump eminently on this new release and install it.

Reply:
Alright, I'll go with it, particularly because 5.16 is the intent here. With the advent of the plugin marketplace I suppose it'll becomes less relevant of an issue going forward since most customers will be installing from there, and presumably we wouldn't serve an incompatible version anyway.